src/resources/keystone/openstack-rc.js

```diff
@@ -36,7 +36,7 @@ export const getPwdOpenRc = (data) => {
     '# OpenStack API is version 3. For example, your cloud provider may implement\n' +
     '# Image API v1.1, Block Storage API v2, and Compute API v2.0. OS_AUTH_URL is\n' +
     '# only for the Identity API served through keystone.\n' +
-    `export OS_AUTH_URL=${authUrl}/v3/\n` +
+    `export OS_AUTH_URL=${authUrl}/v3\n` +
```

Comment:
To resolve this issue, please remove "v3/" from both lines. Afterwards, run "yarn run build" to generate the bundle files that should replace the existing ones.